stock_request_tier_validation/tests/test_stock_request_tier_validation.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017-2020 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+from odoo.tests import common
+
+
+class TestStockRequest(common.TransactionCase):
+    def setUp(self):
+        super(TestStockRequest, self).setUp()
```

Comment:
I think you should change the setUp method to setUpClass according to the [Migration Rules to v15](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0).

![image](https://github.com/OCA/stock-logistics-warehouse/assets/123451545/36e71e9b-be78-4556-985e-bc84a8dd684d)